src/index.html

```diff
@@ -65,5 +70,214 @@ <h1 class="headline__title">
         </li>
       </ul>
     </section>
+    <section class="works">
+      <div class="project-card">
+        <img
+          class="project-card__image"
+          src="./assets/images/snapshoot-1.jpg"
+          alt="Snapshoot 1"
+        />
+        <div class="project-card__body">
+          <div class="project-card__header">
+            <h2 class="project-card__header-title">Tonic</h2>
+            <div class="project-card__header-subtitle">
+              <span>CANOPY</span>
+              <span>Back End Dev</span>
+              <span>2015</span>
+            </div>
+          </div>
+          <p class="project-card__description">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="project-card__tags-list">
+            <li class="project-card__tags-list-item">html</li>
+            <li class="project-card__tags-list-item">css</li>
+            <li class="project-card__tags-list-item">javascript</li>
+          </ul>
+          <div class="project-card__actions">
+            <button class="button" type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+      <div class="project-card">
+        <img
+          class="project-card__image"
+          src="./assets/images/snapshoot-2.jpg"
+          alt="Snapshoot 1"
+        />
+        <div class="project-card__body">
+          <div class="project-card__header">
+            <h2 class="project-card__header-title">Multi-PostStories</h2>
```

Comment:
- [ ] [Optional] Just a small typo here in Multi-Post Stories. I saw it when I was looking at your design so thought I should let you know.

Reply:
### Thank you for your comment.
issue fixed here [Fix Typo in Home page](https://github.com/teckim/portfolio/pull/2/commits/e337d177aaac83ff5a8f873a4db8549c05c6aadf)